autoyast/iPXE.erb

```diff
@@ -0,0 +1,65 @@
+#!gpxe
+<%#
+kind: iPXE
+name: yast default iPXE
+oses:
+- SLES 11
+%>
+<% subnet = @host.subnet -%>
+<% if subnet.respond_to?(:dhcp_boot_mode?) && subnet.dhcp_boot_mode? -%>
+  <% static = '' -%>
+<% else -%>
+  <% static_arg = 'static=yes' -%>
+  <% static = (@host.token.nil? ? '?' : '&') + static_arg -%>
+<% end -%>
+
+:MENU
+menu Choose an action
+item static Install with static ip address (Infos system for more informations)
+item dhcp Install with dhcp
+item reseau Configure network
+item hostinfo Infos system
+item shell Shell access
+item reboot Reboot
+item shutdown Shutdown
+choose target && goto ${target}
+
+
+:static
+kernel <%=@host.url_for_boot(:kernel) %> splash=silent hostip=${ip}/24 gateway=${gateway} install=<%=@host.os.medium_uri(@host)%>  autoyast=<%= foreman_url('provision') %> text-mode=0 domain=<%=@host.domain %> Nameserver=<%=@host.subnet.dns_primary%> netsetup=hostip UseDHCP=0
```

Comment:
Sorry for the late response. I had hardcoded ``/24`` because the static way of booting with my menu was attended to be used with the "Configure Network" option, and after a user would change the mask with the configure option, the mask would no longer be the one from the Foreman. Plus, iPXE does not provide a cidr variable for the mask (e.g ``/24``), only the traditional way `${netmask}`. So I had hard coded the value